form_modify_choice.php

```diff
@@ -90,12 +98,27 @@ public function definition() {
             null, null, array(0, 1));
         $mform->addHelpButton($elementname, 'choice_active', ratingallocate_MOD_NAME);
 
+        $elementname = 'usegroups';
+        $mform->addelement('advcheckbox', $elementname, get_string('choice_usegroups', ratingallocate_MOD_NAME),
+            null, null, array(0, 1));
+        $mform->addHelpButton($elementname, 'choice_usegroups', ratingallocate_MOD_NAME);
+
+        $elementname = 'groupselector';
+        $options = $this->ratingallocate->get_group_selections();
+        $selector = $mform->addelement('searchableselector', $elementname,
```

Comment:
can we please add a $mform->hideif() here so that when the "restrict visibility by groups" is not set, then the groupselector field is hidden? - it shouldn't be possible to add groups when "restrict visibility" is not set.

Reply:
Certainly, done in 981055b.